### PR TITLE
Fix/sample app

### PR DESCRIPTION
**Pull request by adimiz1**
### Brief Summary of Changes
Fix deprecations around the SDK

#### What does this PR address?
- [ ] GitHub issue (Add reference - #XX)
- [ ] Refactoring
- [ ] New feature
- [x] Bug fix
- [ ] Adds more tests

#### Are tests included?
- [x] Yes
- [ ] No

#### Checklist:
- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [x] I ran the full test suite before pushing the changes and all the tests pass.

### CODE REVIEW

File: sample/src/main/java/com/cloudinary/android/sample/app/ResourcesAdapter.java [hunk 1]
```diff
@@ -261,7 +263,11 @@ private void setProgress(ResourceViewHolder holder, ResourceWithMeta resourceWit
 
         String text = holder.statusText.getContext().getString(R.string.uploading, progressStr);
         SpannableString spannableString = new SpannableString(text);
-        spannableString.setSpan(new ForegroundColorSpan(holder.statusText.getContext().getResources().getColor(R.color.buttonColor)), text.indexOf(progressStr), text.length(), 0);
+        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.M) {
+            spannableString.setSpan(new ForegroundColorSpan(holder.statusText.getContext().getColor(R.color.buttonColor)), text.indexOf(progressStr), text.length(), 0);
```
Comment (const-cloudinary) [inline sample/src/main/java/com/cloudinary/android/sample/app/ResourcesAdapter.java:267]: why don't you just extract `holder.statusText.getContext().getColor(R.color.buttonColor))` to a variable ? and reuse the rest of the code?

File: preprocess/src/main/java/com/cloudinary/android/preprocess/BitmapEncoder.java [hunk 2]
```diff
@@ -80,10 +81,15 @@ protected final String saveFile(Context context, Bitmap resource, int quality, F
...
     }
 
+    @SuppressWarnings("deprecation")
     private Bitmap.CompressFormat adaptFormat(Format format) {
         switch (format) {
             case WEBP:
-                return Bitmap.CompressFormat.WEBP;
+                if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.R) {
+                    return Bitmap.CompressFormat.WEBP_LOSSY;
+                } else {
```
Comment (const-cloudinary) [inline preprocess/src/main/java/com/cloudinary/android/preprocess/BitmapEncoder.java:90]: you don't need `else` here

File: sample/src/main/java/com/cloudinary/android/sample/app/Utils.java [hunk 3]
```diff
@@ -94,11 +95,16 @@ public static Uri toUri(Context context, int resourceId) {
...
 
+    @SuppressWarnings("deprecation")
     public static int getScreenWidth(Context context) {
         WindowManager window = (WindowManager) context.getSystemService(Context.WINDOW_SERVICE);
         Point point = new Point();
-        window.getDefaultDisplay().getSize(point);
-        return point.x;
+        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.R) {
+            return window.getCurrentWindowMetrics().getBounds().width();
+        } else {
```
Comment (const-cloudinary) [inline sample/src/main/java/com/cloudinary/android/sample/app/Utils.java:104]: no need for `else`

**Review by const-cloudinary — CHANGES_REQUESTED** [review]
good PR. Please fix a few very minor issues.

File: sample/src/main/java/com/cloudinary/android/sample/app/ResourcesAdapter.java (hunk 1 above)
Comment (adimiz1) [reply sample/src/main/java/com/cloudinary/android/sample/app/ResourcesAdapter.java:267]: Becase getColor from getResources is deprecated, and from getContext is supported in Android M+

File: sample/src/main/java/com/cloudinary/android/sample/app/Utils.java (hunk 3 above)
Comment (adimiz1) [reply sample/src/main/java/com/cloudinary/android/sample/app/Utils.java:104]: The else is for Old android version < Android R

**Review by const-cloudinary — APPROVED** [review]

**Pull request merged**